### PR TITLE
STORM-912: Support SSL on Logviewer

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

**Comment by revans2** [comment]
+1 looks good to me

File: storm-core/src/clj/backtype/storm/ui/core.clj [hunk 1]
```diff
@@ -293,10 +293,17 @@
               (bolt-comp-summs id))]
     (sort-by #(-> ^ExecutorSummary % .get_executor_info .get_task_start) ret)))
 
-(defn worker-log-link [host port topology-id]
+(defn worker-log-link [host port topology-id secure?]
   (let [fname (logs-filename topology-id port)]
-    (url-format (str "http://%s:%s/log?file=%s")
-          host (*STORM-CONF* LOGVIEWER-PORT) fname)))
+    (if secure?
```
Comment (revans2) [inline storm-core/src/clj/backtype/storm/ui/core.clj]: Actually I just had a thought.  We probably want this to be 
```
(if (and secure? (*STORM-CONF* LOGVIEWER-PORT))
...)
```

That way someone is not required to secure their logviewers if they secure the main UI.

**Comment by knusbaum** [comment]
Fixed the boolean condition.

**Comment by revans2** [comment]
looks good to me I am +1 for merging this in, and CI actually passed this time.

**Pull request merged**